bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -93,10 +102,24 @@ GradleBuilder.prototype.prepBuildFiles = function() {
     // Update dependencies within build.gradle.
     var buildGradle = fs.readFileSync(path.join(this.root, 'build.gradle'), 'utf8');
     var depsList = '';
+    var root = this.root;
+    var insertExclude = function(p) {
+          var gradlePath = path.join(root, p, 'build.gradle');
+          var projectGradleFile = fs.readFileSync(gradlePath, 'utf-8');
+          if(projectGradleFile.indexOf('CordovaLib') != -1) {
```

Comment:
It should be safe to add the exclude, since CordovaLib itself will be included in the project.  We require this exclusion because if you make plugins modules, you have to include the CordovaLib file somehow for Android Studio to work properly.  In the near future, we should just use the AAR and not have to deal with this slower solution.  However, it's good to have this just in case.